app/assets/javascripts/auth/cover.js

```diff
@@ -0,0 +1,8 @@
+$(document).ready(function () {
+  var rndNum = Math.floor((Math.random() * 2) + 1);
+
+  if (!$('body section.sign-up, body section.login').length) {
```

Comment:
To improve efficiency this IF should be the first execution line of the method. The rndNum var is only used after the IF.